src/assets/scripts/client/ui/SettingsController.js

```diff
@@ -127,4 +130,24 @@ export default class SettingsController {
 
         return `<option value="${optionData.value}">${optionData.displayLabel}</option>`;
     }
+
+    /**
+     * Build the html for the simulator version psuedo-option.
+     *
+     * @for SettingsController
+     * @method _buildVersionTemplate
+     * @return {JQuery|HTML element}
+     */
+    _buildVersionTemplate() {
+        const simulatorVersion = window.GLOBAL.VERSION;
+        const $container = $(UI_OPTION_CONTAINER_TEMPLATE);
+        const $version = $(UI_OPTION_SELECTOR_TEMPLATE);
+        
+        $version.text(simulatorVersion);
+        $version.addClass('simulator-version');
```

Comment:
the css class should be part of the template. Also, I would prefer a class name like `option-static-text` or something. Maybe we are going to add more text someday.